src/main/java/sentinel/ui/Ui.java

```diff
@@ -172,6 +172,20 @@ public void showList(SentinelList lst) {
         }
     }
 
+    /**
+     * Displays the filtered list of tasks.
+     *
+     * @param lst The list of tasks to filter from.
+     */
+    public void showFileredList(SentinelList lst, String keyword) {
```

Comment:
I like how you created a method dedicated to showing filtered list instead of simply reusing the usual show list method.